examples/producer-random/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/google/uuid"
+	kafka "github.com/segmentio/kafka-go"
+)
+
+func getKafkaWriter(kafkaURL, topic string) *kafka.Writer {
```

Comment:
```suggestion
func newKafkaWriter(kafkaURL, topic string) *kafka.Writer {
```
It looks like this is more of a constructor than a getter.